nautobot_ssot/jobs/examples.py

```diff
@@ -99,6 +106,39 @@ class RoleModel(NautobotModel):
     pk: Optional[UUID]
 
 
+class StatusModel(NautobotModel):
+    """Shared data model representing a Status in either of the local or remote Nautobot instances."""
+
+    # Metadata about this model
+    _model = Status
+    _modelname = "status"
+    _identifiers = ("name",)
+    _attributes = ("content_types", "color")
+
+    name: str
+    color: str
+    content_types: List[ContentTypeDict] = []
+
+    # Not in _attributes or _identifiers, hence not included in diff calculations
+    pk: Optional[UUID]
```

Comment:
This doesn't need to be explicitly defined, its already in `NautobotModel`

Reply:
Alright, will remove. I feel like those were there prior to my updates but no worries.

Reply:
Yes they were - but since you added some I thought I'd mention it

Reply:
Fair enough. Was just replicating the others. I've removed it from all the models though.